examples/joystick/Joystick.cpp

```diff
@@ -24,25 +24,25 @@ Texts              texts;
 std::ostringstream sstr;
 float              threshold = 0.1f;
 
-// Axes labels in as C strings
-constexpr std::array axislabels = {"X", "Y", "Z", "R", "U", "V", "PovX", "PovY"};
+// Axes labels in as strings
+const std::array<std::string, 8> axislabels = {"X", "Y", "Z", "R", "U", "V", "PovX", "PovY"};
 
 // Helper to set text entries to a specified value
 template <typename T>
-void set(const char* label, const T& value)
+void set(const std::string& label, const T& value)
 {
     sstr.str("");
     sstr << value;
-    texts[label].value.setString(sstr.str());
+    texts.at(label).value.setString(sstr.str());
 }
 
 // Update joystick identification
 void updateIdentification(unsigned int index)
 {
     sstr.str("");
     sstr << "Joystick " << index << ":";
-    texts["ID"].label.setString(sstr.str());
-    texts["ID"].value.setString(sf::Joystick::getIdentification(index).name);
+    texts.at("ID").label.setString(sstr.str());
+    texts.at("ID").value.setString(sf::Joystick::getIdentification(index).name);
```

Comment:
Avoid a double lookup:

```suggestion
    auto& [label, value] = texts.at("ID");
    label.setString(sstr.str());
    value.setString(sf::Joystick::getIdentification(index).name);
```